topologies/datacenter-latest/labguides/source/cvp-vxlan.rst

```diff
@@ -98,63 +123,63 @@ VxLAN
     :align: center
     :width: 50% 
 
-* Click **save** on the Network Provisioning main view
-    .. note: a **Task** will be generated
+* Click **Save** on the **Network Provisioning** main view
+   **Note:** a **Task** will be generated
 
 .. image:: images/cvp-vxlan/leaf3-vxlan-configlet-main-save.png
     :align: center
     :width: 50% 
 
 5. Create a **Change Control** with the generated Task
 
-* click **Tasks** from the left selection column
+* Click **Tasks** from the left selection column
 
-* click the checkbox next to the generated task
+* Click the checkbox next to the generated task from the pool of **Assignable Tasks**
 
 .. image:: images/cvp-vxlan/leaf3-vxlan-cc-task.png
     :align: center
     :width: 50% 
 
-* click **+** Create Change Control with 1 Task
+* Click **+** Create Change Control with 1 Task
 
 .. image:: images/cvp-vxlan/leaf3-vxlan-cc-create-cc.png
     :align: center
     :width: 50% 
 
-* click **Review and Approve** on the Change Control that was created
+* Click **Review and Approve** on the newly created **Change Control**
 
 .. image:: images/cvp-vxlan/leaf3-vxlan-cc-review-approve.png
     :align: center
     :width: 50% 
 
-* click **Execute Change Control** in upper right of the UI
+* Click **Execute Change Control** in upper right of the UI
```

Comment:
```suggestion
* Click **Execute Change Control** towards the upper right of the UI
```

Reply:
I am going to reject this suggestion, I think it is even more vague by adding "towards"